dist/profile/manifests/kubernetes/delete.pp

```diff
@@ -29,24 +31,34 @@
   $basename = basename($resource)
 
   # Add resource file to trash directory
-  file { "${profile::kubernetes::params::trash}/${dirname}.${basename}":
+  file { "${profile::kubernetes::params::trash}/${context}.${dirname}.${basename}":
     ensure  => 'present',
     content => template("${module_name}/kubernetes/resources/${resource}.erb"),
     owner   => $profile::kubernetes::params::user
   }
 
-  $clusters.each | $cluster | {
-    # Only run kubectl delete if the resources is deployed.
-    exec { "Remove ${resource} on ${cluster[clustername]}":
-      command     => "kubectl delete --grace-period=60 --ignore-not-found=true -f ${profile::kubernetes::params::trash}/${dirname}.${basename}",
-      path        => [$profile::kubernetes::params::bin,$::path],
-      environment => ["KUBECONFIG=${profile::kubernetes::params::home}/.kube/${cluster[clustername]}.conf"] ,
-      onlyif      => "test \"$(kubectl apply --dry-run=true -f ${profile::kubernetes::params::trash}/${dirname}.${basename} | grep configured)\""
-    }
+$delete_args = "\
```

Comment:
I'm surprised this variable declaration and `$apply_args` are passing Puppet Lint. Please indent them to the appropriate level within the resource.